lib/xcode_summary/plugin.rb

```diff
@@ -209,14 +209,18 @@ def parse_test_location(failure)
 
     def format_path(path)
       clean_path, line = parse_filename(path)
-      path = clean_path + '#L' + line if clean_path && line
+      path = clean_path + ':' + line if clean_path && line
 
       # Pick a Dangerfile plugin for a chosen request_source
       # based on https://github.com/danger/danger/blob/master/lib/danger/plugin_support/plugin.rb#L31
       plugins = Plugin.all_plugins.select { |plugin| Dangerfile.essential_plugin_classes.include? plugin }
       plugin = plugins.select { |p| p.method_defined? :html_link }.map { |p| p.new(@dangerfile) }.compact.first
 
-      plugin.html_link(path)
+      if plugin
```

Comment:
Good catch!
The idea here is to add the link to the path if it's possible, given that we don't have this here it should better to return the original `path`, so I suggest:

```ruby
    def format_path(path)
      # Pick a Dangerfile plugin for a chosen request_source
      # based on https://github.com/danger/danger/blob/master/lib/danger/plugin_support/plugin.rb#L31
      plugins = Plugin.all_plugins.select { |plugin| Dangerfile.essential_plugin_classes.include? plugin }
      plugin = plugins.select { |p| p.method_defined? :html_link }.map { |p| p.new(@dangerfile) }.compact.first

      if plugin
        clean_path, line = parse_filename(path)
        path = clean_path + '#L' + line if clean_path && line
        plugin.html_link(path)
      else
        path
      end
    end
```